hca/util/exceptions.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from requests.exceptions import HTTPError
+
+class SwaggerAPIException(HTTPError):
+    """
+    Exception raised by SwaggerClient when an HTTP error is received from a Swagger API server.
+    """
+    def __init__(self, *args, **kwargs):
+        super(SwaggerAPIException, self).__init__(*args, **kwargs)
+
+        self.code = self.response.status_code
```

Comment:
there's a standardized template for responses that comes as part of the json.  https://github.com/HumanCellAtlas/data-store/blob/b9471df71eeba3074a43ca50caee5c0e144f0f15/dss-api.yml#L998

Reply:
Added error response fields to exception.